recipes/dcv_install.rb

```diff
@@ -94,6 +94,7 @@ def disable_lock_screen
         cwd Chef::Config[:file_cache_path]
         code <<-PREREQ
           set -e
+          apt -y install whoopsie
```

Comment:
can these two lines be a one-liner?

Reply:
I did it on separate lines because I didn't want apt to see that whoopsie would be installed as part of ubuntu-desktop anyways, and then not install it first. That would make it effectively the same as it is now. I'm not sure if that's actually how it would work.